exercises/reforestation/canonical-data.json

```diff
@@ -0,0 +1,93 @@
+{
+    "exercise": "reforestation",
+    "version": "1.0.0",
+    "cases": [
+      {
+        "description": "Base case: empty tree",
+        "property": "treeTraversals",
```

Comment:
I'm not sure about the property name. Yes one has to do a tree traversal, but that is the method and not the goal I think. Maybe something like "treeFromTraversals"?

Reply:
I chose the property name to be idiomatic with Prolog (tree_traversals/3). I like your suggestion.